config/settings/base.py

```diff
@@ -223,3 +223,5 @@
     # something more human-readable.
     # release="myapp@1.0.0",
 )
+
```

Comment:
Is this localhost url  on line 227 valid in the cluster this is hosted on?

Reply:
Probably doesn't matter as thats just the default if it can't find the env var.

Reply:
Yeah just the default for tests and things, I will set the env where I deploy it